tests/data/PA1_expected_0_i.yaml

```diff
@@ -0,0 +1,11 @@
+!<lsst.verify.Measurement>
```

Comment:
Should this (and the other photometry metric files) be PA1_expected_0_70_*.yaml? (I'm curious why it's inconsistent with the others.)

Reply:
Oh, never mind - I see now that you persisted tract-level catalogs along with the patch ones.